src/views/ManageList.jsx

```diff
@@ -2,29 +2,49 @@ import { addItem } from '../api/firebase';
 import { useState } from 'react';
 import ShareForm from '../components/ShareForm';
 
-export function ManageList({ listPath }) {
+export function ManageList({ listPath, data }) {
```

Comment:
big bonus: write a test that tests this components logic could create an issue to look into this later.

Reply:
Hi @bdanzer, I'm creating tests for the AddItemForm component now and just wanted to make sure I'm on the right track - when testing this form I would have to first create a test user, then create a test list, then finally test adding items?

Reply:
> Hi @bdanzer, I'm creating tests for the AddItemForm component now and just wanted to make sure I'm on the right track - when testing this form I would have to first create a test user, then create a test list, then finally test adding items? 

@jtkabenni that is correct, you'll also mock the add item call too.

Reply:
@bdanzer I think I may still be confused on what mocking would look like here. Am I making calls to the database to create the user and list, or just providing the data in the test? And if i am mocking the add item call, how would I confirm that the function is working as expected?